app/notify_admin_test.go

```diff
@@ -17,7 +17,7 @@ import (
 
 func Test_SendNotifyAdminPosts(t *testing.T) {
 
-	t.Run("no error sending upgrade post when no notifications are available", func(t *testing.T) {
+	t.Run("no error sending non trial post when no notifications are available", func(t *testing.T) {
```

Comment:
I'd highly recommend a test where the mixed types of notify admin requests are sent, and validating that the user specific information of the post is what is expected (i.e. you're not accidentally mixing work templates information with feature request info)